requirements-dev.txt

```
@@ -0,0 +1,2 @@
+colorama==0.4.6
```

:green_circle: Importer le contenu de `requirements.txt` pour n'avoir qu'à installer celui-ci en local
```suggestion
-r requirements.txt
colorama==0.4.6
```